server/src/graphql/resolvers/mutations.js

```diff
@@ -15,4 +16,28 @@ module.exports = {
     const user = await User.create({ id: uuidv4(), email, password: hash });
     return user ? true : false;
   },
+
+  upsertWell: async (_, { model }) => {
+    if (!model.id) {
+      model.id = uuidv4();
+    }
+    await Well.upsert({ ...model });
+    return Well.findByPk(model.id);
+  },
+
+  deleteWell: async (_, { id }) => {
+    const deletedWell = await Well.findOne({ where: { id } });
+    await Well.destroy({ where: { id } });
+    return deletedWell;
+  },
+
+  deleteWells: async (_, { wellIds }) => {
+    const deletedWells = wellIds.map(
```

Comment:
same thing here. If you already have the ids available, just return a list of ids if the deletion is successful.